core/src/language/swift.rs

```diff
@@ -755,6 +768,35 @@ impl Swift {
         decs.extend(self.default_decorators.iter().cloned());
         decs
     }
+
+    /// When using mulitple file generation we write this into a separate module vs at the
+    /// end of the generated file.
+    fn write_codable_file(&self, output_folder: &str) -> std::io::Result<()> {
+        let mut w = OpenOptions::new()
+            .write(true)
+            .create(true)
+            .truncate(true)
+            .open(PathBuf::from(output_folder).join("Codable.swift"))?;
```

Comment:
Can reduce to `File::create`

nit: L779: Can `Path::from` to save 1 aloc

Reply:
Updated